lib/types.ts

```diff
@@ -237,10 +237,10 @@ export type TemplateImageCarousel = {
 
 export type TemplateImageColumn = {
   imageUrl: string;
-  action: TemplateAction<{ label?: string }>;
+  action: Action<{ label?: string }>;
 };
 
-export type TemplateAction<Label> =
+export type Action<Label> =
   | TemplatePostbackAction & Label
```

Comment:
Please remove `Template-` prefixes of summed types too, e.g. `TemplatePostbackAction` → `PostbackAction`.